common/tables/table-en.html

```diff
@@ -0,0 +1,622 @@
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+			<tr>
+				<th>2</th>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+			<tr>
+				<th>3</th>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+				<td>Table cell</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<pre><code>&lt;div class="table-responsive"&gt;
+&lt;table class="table"&gt;
+...
+&lt;/table&gt;
+&lt;/div&gt;</code></pre>
+
+<h2>Table Grids <code>.tbl-gridify</code></h2>
+<table class="table tbl-gridify">
+	<thead>
+		<tr>
+			<th>Number</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+			<th>Column heading</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr class="active">
+			<th>1</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>2</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="success">
+			<th>3</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>4</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="info">
+			<th>5</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>6</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="warning">
+			<th>7</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr>
+			<th>8</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+		<tr class="danger">
+			<th>9</th>
+			<td>Column content</td>
+			<td>Column content</td>
+			<td>Column content</td>
+		</tr>
+	</tbody>
+</table>
+<pre><code>&lt;table class="table tbl-gridify"&gt;
+...
+&lt;/table&gt;</code></pre>
+
+<h2> Tables within panels </h2>
+<p>Full-width content:</p>
+	<div class="panel panel-default">
+		<table class="table">
+		<caption class="wb-inv">Caption text</caption>
+		<thead>
+			<tr>
+				<th>Table header</th>
+				<th>Table header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<p>Padded plus full-width content:</p>
+<div class="panel panel-default">
+	<div class="panel-body">
+		<p>Panel body Content</p>
+	</div>
+	<table class="table">
+		<caption class="wb-inv">
+			Caption  text
+		</caption>
+		<thead>
+			<tr>
+				<th>Table header</th>
+				<th>Table header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+			<tr>
+				<td>Table data</td>
+				<td>Table data</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
```

Comment:
I do know this example are coming from the style guide but this is not a valid panel example, the body marker must wrap all the panel content

```suggestion
```

Reply:
This was addressed